src/lang/ts/intermock.ts

```diff
@@ -13,10 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import ts from 'typescript';
+import ts, {ArrayTypeNode, FunctionTypeNode, isArrayTypeNode, isFunctionTypeNode, isTypeNode, LiteralTypeNode, NumericLiteral, TypeReferenceNode} from 'typescript';
```

Comment:
Probably best to take all of `ts` or only specific exports. You could probably just remove these specific ones and just reference off of `ts`.

Reply:
Done